### PR TITLE
Added Timeline-Track Functionality

### LINKED ISSUE #114: Integrate Timeline Track

### PR DESCRIPTION
**Pull request by javitrejo14**
Closes #114

Slider moves according to current frame and manual timeline. Slider can now also control current frame.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -19,6 +19,8 @@ export default function App() {
 	const [playing, setPlaying] = useState(false);
 	const [frameStart] = useState();
 	const [frameEnd, setEnd] = useState(500);
+
+	const [value, setValue] = useState(0);
```
Comment (vpzomtrrfrt) [inline src/App.tsx:23]: what is this?

File: package-lock.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-	"name": "f22-viz-kids-1",
+	"name": "f22-viz-kids",
```
Comment (vpzomtrrfrt) [inline package-lock.json:2]: unrelated change

**Review by vpzomtrrfrt — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -146,7 +148,7 @@ export default function App() {
 				<button id={"play-button"} onClick={togglePlaying}>
 					{playing ? <PauseIcon /> : <PlayIcon />}
 				</button>
-				<input id={"timeline-track"} type={"range"} min={"0"} max={"494"} value={"0"} />
+				<input id={"timeline-track"} type={"range"} min={"0"} max={"494"} onChange={(e) => setFrame(parseInt(e.target.value))} value={frame} />
```
Comment (PTLux1517) [inline src/App.tsx:151]: For this to work with files other than our sample data, the max attribute will need to be set to markerFileData.frames.length - 1

File: src/App.tsx (hunk 1 above)
Comment (javitrejo14) [reply src/App.tsx:23]: I was using this before but ended up not needing it. Removed now.

File: package-lock.json (hunk 2 above)
Comment (javitrejo14) [reply package-lock.json:2]: Removed

File: src/App.tsx [hunk 4]
```diff
@@ -146,7 +146,8 @@ export default function App() {
 				<button id={"play-button"} onClick={togglePlaying}>
 					{playing ? <PauseIcon /> : <PlayIcon />}
 				</button>
-				<input id={"timeline-track"} type={"range"} min={"0"} max={"494"} value={"0"} />
+				<input id={"timeline-track"} type={"range"} min={"0"} max={markerFileData.frames.length - 1}
+				 onChange={(e) => setFrame(parseInt(e.target.value))} value={frame} />
```
Comment (vpzomtrrfrt) [inline src/App.tsx:150]: should be indented with tabs, not a space

**Review by vpzomtrrfrt — CHANGES_REQUESTED** [review]

**Review by vpzomtrrfrt — APPROVED** [review]

**Review by PTLux1517 — APPROVED** [review]
